### PR TITLE
Update Episode Route to Use Episode

### PR DESCRIPTION
**Pull request by stephencelis**

### CODE REVIEW

**Comment by stephencelis** [comment]
This isn't ready...some of the mock stuff made it more complicated, so let's chat about it.

**Comment by stephencelis** [comment]
One failing test around episode credits that I haven't quite figured out yet but my brain's a bit zonked...

File: Sources/PointFree/Episode/Show.swift [hunk 1]
```diff
@@ -13,15 +13,18 @@ import Styleguide
...
+  = writeStatus(.ok)
     >=> userEpisodePermission
     >=> map(lower)
     >>> respond(
       view: Views.episodeView(episodePageData:),
       layoutData: { permission, episode, currentUser, subscriberState, currentRoute in
         let navStyle: NavStyle = currentUser == nil ? .mountains(.main) : .minimal(.light)
+        #if DEBUG
+        var episode = Current.episodes().first(where: { $0.sequence == episode.sequence }) ?? episode
+        #endif
```
Comment (stephencelis) [inline Sources/PointFree/Episode/Show.swift]: These lines are necessary for routing to go through tests. You might have a better idea...

Comment (mbrandonw) [reply Sources/PointFree/Episode/Show.swift]: one thought comes to mind, we could inject the episodes into the router here: https://github.com/pointfreeco/pointfreeco/blob/master/Sources/PointFree/Bootstrap.swift#L88

**Pull request closed without merging**